src/models/connectionStore.ts

```diff
@@ -632,6 +634,62 @@ export class ConnectionStore {
         return output;
     }
 
+    /**
+     * Get the options for connection groups.
+     * @returns A promise that resolves to an array of FormItemOptions for connection groups.
+     */
+    public async getConnectionGroupOptions(): Promise<FormItemOptions[]> {
```

Comment:
Commented earlier, but maybe connectionUi.ts or connectionManager are more appropriate places for this since it doesn't work directly with the config JSON.